glide.yaml

```diff
@@ -0,0 +1,21 @@
+package: github.com/Azure/azure-sdk-for-go
+import:
+- package: github.com/Azure/azure-sdk-for-go
+  subpackages:
+  - /arm/examples/helpers
+  - arm/storage
+  - core/http
+  - core/tls
+  - management
+  - management/location
+  - management/virtualmachine
+  - management/virtualmachinedisk
```

Comment:
I need to take a look at this more closely, I don't know why it's even listing itself in here.